src/thunderbots/software/test/primitive/directvelocity_primitive.cpp

```diff
@@ -0,0 +1,131 @@
+/**
```

Comment:
Looks like we forgot to remove this file as well. Unit tests for the direct velocity primitive already exist so this file can be removed.